content/release-notes/1.59.1.md

```diff
@@ -0,0 +1,10 @@
+---
+date: 2021-12-29T10:00:00-05:00
+linktitle: "1.59.1"
+title: "1.59.1"
+kubernetes: "1.19, 1.20, and 1.21"
+---
+
+{{<fixes>}}
+* Fixes a panic that happens when [kots upstream upgrade](/kots-cli/upstream/) command.
```

Comment:
```suggestion
* Fixes an error that occurs when the [`kots upstream upgrade`](/kots-cli/upstream/) command is run.
```

Reply:
Was there a specific error that was thrown when running the `kots upstream upgrade` command that is now fixed? Just looking to add a little more clarity to what we fixed with this release for the end-user :)

Reply:
Per slack, this is the error: "$ kdev kots upstream upgrade dmitriy-kots -n dmitriy
  • Checking for application updates ⠹panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0x2ee9789]"